mlfinlab/bet_sizing/targeting.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains functionality for determining bet sizes for investments based on volatility targeting.
+"""
+
+import numpy as np
+
+
+def volatility_targeting(returns: np.ndarray, target_vol: float, annualize_factor: int):
+    """
+    Calculate the position size to make the expected volatility of the portfolio be the target variability.
+
```

Comment:
Is there a reference to this method (paper/book)?

Reply:
It's my mistake. I thought this is kind of general skills. I'll find the reference paper.